db/migrate/20160310044144_add_doc_id_to_topic.rb

```diff
@@ -0,0 +1,26 @@
+class AddDocIdToTopic < ActiveRecord::Migration
+
+  def up
+    add_column :topics, :doc_id, :integer, default: 0
+    add_column :docs, :topics_count, :integer, default: 0
+    add_column :docs, :allow_comments, :boolean, default: true
+
+    if Forum.where(name: 'Doc comments').first.nil?
```

Comment:
Okay, that makes sense I suppose. Does the forum contain _all_ comments for _all_ documents or something? I don't fully understand the use-case for this, but agree that if we need this it'd seem there's no better place for it than in a migration.